rocsp/config/rocsp_config.go

```diff
@@ -25,8 +25,15 @@ type RedisConfig struct {
 	TLS cmd.TLSConfig
 	// Username is a Redis username.
 	Username string
-	// Addrs is a list of IP address:port pairs.
+	// Addrs is a list of IP address:port pairs. The go-redis `ClusterClient`
+	// will use this list to discover a cluster of Redis servers configured in
+	// Cluster mode. DEPRECATED: Use `ShardAddrs` instead.
```

Comment:
Put the "DEPRECATED: ..." notice [in its own paragraph](https://go.dev/blog/godoc#:~:text=To%20signal%20that%20an%20identifier%20should%20not%20be%20used%2C%20add%20a%20paragraph%20to%20its%20doc%20comment%20that%20begins%20with%20%E2%80%9CDeprecated%3A%E2%80%9D%20followed%20by%20some%20information%20about%20the%20deprecation.), so that go's automated tooling can pick up on it.